### PR TITLE
Created preferences object with very basic UI for now

### PR DESCRIPTION
**Pull request by kkindorf**

### CODE REVIEW

**Comment by drerichards** [comment]
Hello Kevin,

I'm by no means an expert at this, but I'd say the setup of your JS file was well done. My eyes did not have to do much darting to figure out the flow of your program. It's in the beginning stages so the need for an elaborate UI isn't necessary until the back end is tackled. So that helped to ease readability of your code. My only two critiques would just be for elaboration in your comments about the actual purpose or intent of your functions. And secondly, I'd opt for more consistency in capitalization with your flavors and descriptions. Other than those minor things, you're well on your way!

File: js/app.js [hunk 1]
```diff
@@ -1,63 +1,111 @@
...
-	flavor: [description, description, description]
-	flavor: [description, description, description]
-	flavor: [description, description, description]
-	flavor: [description, description, description]
-}*/
-
-/*questions constructor wth inherited questions*/
-
-var question = function(flavor, text){
+Bartender.prototype.showQuestions = function(questions){
```
Comment (aric87) [inline js/app.js]: add a comment that lets the next developer know what questions should be, like `@param questions: array[string]`

File: js/app.js [hunk 2]
```diff
@@ -1,63 +1,111 @@
...
+	count = 0;
+  $("#introduction").on("click", "button", function(){
+  	$("#introduction").hide();
+  	$("#questionBox").show();
+  	$("#questionText").html("<p>"+questions[count].text+"</p>")
+  })
+  $("#questionBox").on("click", "button", function(){
```
Comment (aric87) [inline js/app.js]: you reference $('#questionBox') a few times. Yuo should store it in a variable so the browser doesn't have to look it up each time.

File: js/app.js [hunk 3]
```diff
@@ -1,63 +1,111 @@
...
-
-
+/*Pantry constructor*/
+var Pantry = function(ingredients){
+	this.ingredients = ingredients;
+}
+Pantry.prototype.fetchRandom = function(flavor){
+	//create filtered ingredient array 
+	var filteredIngredients = [];
+	for (var i = 0; i < this.ingredients.length; i++){
```
Comment (aric87) [inline js/app.js]: look in to using the reduce method here

File: js/app.js [hunk 4]
```diff
@@ -1,63 +1,111 @@
...
 
 $(function(){
-	
-})
\ No newline at end of file
+	appendQuestions();
+})
+
+function appendQuestions(){
+ var myBartender = new Bartender(questionsArray, myPantry);
```
Comment (aric87) [inline js/app.js]: indentation

File: js/app.js (hunk 3 above)
Comment (jayzamazing) [inline js/app.js]: I would consider storing the data in son file and then bring it in as an object.

File: js/app.js (hunk 1 above)
Comment (jayzamazing) [inline js/app.js]: For the css I tend to use classes over id's, only using id's when it is absolutely necessary.

**Pull request merged**